scrolltoggle/index.js

```diff
@@ -18,11 +23,17 @@ var defaultConfig = {
  */
 function ScrollToggle(el, config) {
   this.el_ = el;
-  // TODO(stevenle): support config overrides per element using a data- attr.
-  this.config_ = config;
-  this.lastScrollPos_ = 0;
+
+  this.config_ = objects.clone(config);
+  if (this.el_.hasAttribute('data-scrolltoggle')) {
```

Comment:
Should we prefix data attrs with `data-ak-` or not? I want to be consistent on the youtubemodal also.